graph/options/options.go

```diff
@@ -179,6 +180,24 @@ func parseAppenders(params url.Values, o Options) []appender.Appender {
 	return appenders
 }
 
+// Gets a map containing all the namespaces available
```

Comment:
The Go convention is to lead the comment with the function name.  Also, maybe describe why this is a map (I assume it's to act like a Set):
`// getAccessibleNamespaces returns a Set of all namespaces accessible to the user. The Set is implemented using the map[string]bool convention.`